### PR TITLE
Upgrade TheTheme to use Bootstrap 5.3.2

### PR DESCRIPTION
**Pull request by MikeAlhayek**
@agriffard TheTheme is now Bootstrap 5.3! If you can please test drive it if you can. I like to merge it soon to give users chances to report issues.

TheTheme now loads Bootstrap independent of the custom CSS. This will make maintainability of the theme easier and also allow the reuse of the same bootstrap package.

Light theme
image

image

Dark Theme
image

### CODE REVIEW

**Comment by agriffard** [comment]
On small screen, please check the theme picker behavior when the user menu is clicked:

Animation

**Comment by MikeAlhayek** [comment]
@agriffard this is the expected behavior in Bootstrap 5. The other option is to use Offcanvas navbar https://getbootstrap.com/docs/5.3/components/navbar/#nav

**Comment by MikeAlhayek** [comment]
@agriffard I improved the alignment. Let me know if you see anything else.

**Comment by agriffard** [comment]
Thank you, it looks better:

image

File: src/OrchardCore.Themes/TheTheme/Views/Layout.cshtml [hunk 1]
```diff
@@ -1,28 +1,39 @@
...
+<html lang="@Orchard.CultureName()" dir="@Orchard.CultureDir()" data-bs-theme="@await ThemeTogglerService.CurrentTheme()" data-tenant="@ThemeTogglerService.CurrentTenant">
 <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>@RenderTitleSegments(Site.SiteName, "before")</title>
-    <link type="image/x-icon" rel="shortcut icon" href="~/TheTheme/favicon.ico">
+    <link type="image/x-icon" rel="shortcut icon" href="~/TheTheme/images/favicon.ico">
+
+    <!-- This script can't wait till the footer -->
+    <script asp-name="admin-head" version="1" at="Head"></script>
```
Comment (agriffard) [inline src/OrchardCore.Themes/TheTheme/Views/Layout.cshtml:23]: Why would TheTheme need `admin` scripts?

Comment (MikeAlhayek) [reply src/OrchardCore.Themes/TheTheme/Views/Layout.cshtml:23]: @agriffard 
The admin-head scripts contains the same code that we use for the toggler and setting theme preferences.

The good thing is that we use the same script once and we cache and reuse it

Comment (agriffard) [reply src/OrchardCore.Themes/TheTheme/Views/Layout.cshtml:23]: Ok but there is no need for the `admin-menu` then, right?

Comment (MikeAlhayek) [reply src/OrchardCore.Themes/TheTheme/Views/Layout.cshtml:23]: No admin-menu is for the admin menu

File: src/OrchardCore.Themes/TheTheme/Views/Layout.cshtml [hunk 2]
```diff
@@ -1,28 +1,39 @@
...
+
     <style asp-name="font-awesome" version="6"></style>
-    <resources type="Meta" />
-    <resources type="HeadLink" />
-    <resources type="Stylesheet" />
-    <resources type="HeadScript" />
+    <resources type="Header" />
     @await RenderSectionAsync("HeadMeta", required: false)
 </head>
 <body dir="@Orchard.CultureDir()">
```
Comment (agriffard) [inline src/OrchardCore.Themes/TheTheme/Views/Layout.cshtml:35]: Remove `dir="@Orchard.CultureDir()"` from `<body>`, now that you added it on `<html>`.

File: src/OrchardCore.Themes/TheTheme/Views/Layout.cshtml [hunk 3]
```diff
@@ -1,28 +1,39 @@
...
 <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>@RenderTitleSegments(Site.SiteName, "before")</title>
-    <link type="image/x-icon" rel="shortcut icon" href="~/TheTheme/favicon.ico">
+    <link type="image/x-icon" rel="shortcut icon" href="~/TheTheme/images/favicon.ico">
+
+    <!-- This script can't wait till the footer -->
+    <script asp-name="admin-head" version="1" at="Head"></script>
+    <script asp-name="admin-menu" depends-on="admin-head" version="1" at="Head"></script>
```
Comment (agriffard) [inline src/OrchardCore.Themes/TheTheme/Views/Layout.cshtml:24]: I mean this `admin-menu`, in TheTheme.
You can remove it, right?

**Comment by agriffard** [comment]
Also tested with `main-menu` with sub items:

image

I'd prefer to have the Menu next to the Site title, as I had managed to do it there:
https://github.com/OrchardCMS/OrchardCore/pull/13524

image

but it might be difficult to remove it from the collapsed div or it won't appear on small screens.

File: src/OrchardCore.Themes/TheTheme/Views/Layout.cshtml (hunk 3 above)
Comment (MikeAlhayek) [reply src/OrchardCore.Themes/TheTheme/Views/Layout.cshtml:24]: Agreed. A copy/paste issue. I'll fix it

**Comment by MikeAlhayek** [comment]
@agriffard I don't think you can put the menu next to the title. We'll technically you can, but if you have couple menu items, it won't look good. Personally, I would not want to make this change. We could shift the menu to the end of the screen instead of next to the title which may line up better on small screen. 

I think we should use the new off canvas menu instead. Anyway, let's not block this PR over preferences. We can also submit another PR to make changes that may require discussion.

**Comment by MikeAlhayek** [comment]
image

image


image

image

image

File: src/OrchardCore.Themes/TheTheme/Views/Layout.cshtml (hunk 2 above)
Comment (MikeAlhayek) [reply src/OrchardCore.Themes/TheTheme/Views/Layout.cshtml]: done

File: src/OrchardCore.Themes/TheTheme/Views/Layout.cshtml (hunk 3 above)
Comment (MikeAlhayek) [reply src/OrchardCore.Themes/TheTheme/Views/Layout.cshtml]: fixed

**Comment by MikeAlhayek** [comment]
@agriffard I like to merge this PR today/tomorrow. Did you get a chance to give it one last test? Any other issues?

**Review by agriffard — APPROVED** [review]
LGTM

**Pull request merged**